tests/api-testing/tests/test_job.py

```
@@ -0,0 +1,219 @@
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)